File: TUGAS 6/TUGAS 6.2 .pas
{Nama    : Fauzy Ramadan}
{NPM     : 20.14.1.0051}
{Program : TUGAS 6.2 .pas}

program TUGAS6_2;
uses crt;

procedure biodata2;
var
nama, wa,  NPM, alamat, TTL, email : string;

jenis_kelamin : char;

begin
write ('Masukan Nama                  = '); readln (nama);
write ('Masukan NPM                   = '); readln (NPM);
write ('Masukan TTL                   = '); readln (TTL);
write ('Masukan Alamat                = '); readln (alamat);
write ('Masukan Jenis Kelamin(L/P)    = '); readln (jenis_kelamin);
write ('Masukan email                 = '); readln (email);
write ('Masukan No Whatsapp           = '); readln (wa);
writeln;
writeln ('Nama Anda          = ', nama);
writeln ('NPM Anda           = ', NPM);
writeln ('TTL Anda           = ', TTL);
writeln ('Alamat Anda        = ', alamat);
writeln ('Jenis Kelamin Anda = ', jenis_kelamin);
writeln ('Email Anda         = ', email);
writeln ('No Whatsapp Anda   = ', wa);
end;

begin
clrscr;
biodata2;

readln;
end.

File: TUGAS 6/TUGAS 6.2.cpp
#include <iostream>
using namespace std;

string nama, NPM, alamat, ttl, wa, email, jk;

void biodata()
{
	cout << "MASUKAN BIODATA ANDA"<<endl;
	cout << "Nama				: " ;
	getline (cin, nama);
	cout << "NPM				: " ;
	getline (cin, NPM);
	cout << "Jenis Kelamin (L/P)		: " ;
	getline (cin, jk);
	cout << "Alamat				: ";
	getline (cin, alamat);
	cout << "TTL				: ";
	getline (cin, ttl);
	cout << "No. WA				: ";
	getline (cin, wa);
	cout << "Email				: ";
	getline (cin, email);
	cout << endl;
	
	cout << "Nama Anda Adalah		: " << nama <<endl;
	cout << "NPM Anda Adalah			: " << NPM <<endl;
	cout << "Jenis Kelamin Anda Adalah	: " << jk <<endl;
	cout << "Alamat Anda Adalah		: " << alamat <<endl;
	cout << "TTL Anda Adalah			: " << ttl <<endl;
	cout << "No. WA Anda Adalah		: " << wa <<endl;
	cout << "Email Anda Adalah		: " << email <<endl;
}

int main()
{
	biodata();
}
